api/index.js

```diff
@@ -1,5 +0,0 @@
-const { createRequestHandler } = require("@remix-run/vercel");
-
-module.exports = createRequestHandler({
-  build: require("./_build")
-});
```

Comment:
So we won't be able to deploy to vercel any more? Was it necessary?

Reply:
That's actually a very valid point, i can keep it. That and the vercel.json

Reply:
But it will never work on vercel because of the database

Reply:
true, hmmm, that means to finish the migration I need to also update the DNS to fly etc, we should examine a bit if our new setup holds a decent spike of visitors if we release the beta and there will be a huge traffic spike

Reply:
going down during beta launch is the worst thing that can happen to a startup

Reply:
I would take a few most database heavy actions and generate workload, I remember from many years ago this tool was great, but maybe there is something better now https://github.com/wg/wrk

Reply:
But the idea is thatr we migrate now and we don't go into beta now

Reply:
What if we go in beta in 5 month and we have to launch with this?

Reply:
But why would we only migrate in 5months? All that's missing is the seeds

Reply:
Nono, I mean lets assume we migrate now and this thing is incapable is serving enough user to survive beta spike, should we try to test it now?